metro.config.js

```diff
@@ -1,9 +1,10 @@
-const { getDefaultConfig } = require("expo/metro-config");
+// const { getDefaultConfig } = require("expo/metro-config");
```

Comment:
Arn't we missing any other values from getDefaultConfig from metro config? Now we only use the sentry expo config as base and later on add extra config items. 

It could be the case that we are missing values from getDefaultConfig now?

Reply:
This function includes the config expo config.

See this:
https://github.com/expo/fyi/blob/main/sentry-expo-migration.md#update-your-metroconfigjs-to-include-the-sentry-transformer-plugin